src/js/__tests__/__snapshots__/tslint-test.ts.snap

```diff
@@ -347,6 +347,11 @@ exports[`tslint snapshot should be up to date 1`] = `
 /plugins/services/src/js/components/forms/NetworkingFormSection.tsx - Shadowed name: 'networks'
 /plugins/services/src/js/components/forms/NetworkingFormSection.tsx - Shadowed name: 'networks'
 /plugins/services/src/js/components/forms/NetworkingFormSection.tsx - don't declare variable selections to return it immediately
+/plugins/services/src/js/components/forms/VolumesFormSection.tsx - Lambdas are forbidden in JSX attributes due to their rendering performance impact
+/plugins/services/src/js/components/forms/VolumesFormSection.tsx - Lambdas are forbidden in JSX attributes due to their rendering performance impact
+/plugins/services/src/js/components/forms/VolumesFormSection.tsx - Lambdas are forbidden in JSX attributes due to their rendering performance impact
+/plugins/services/src/js/components/forms/VolumesFormSection.tsx - Lambdas are forbidden in JSX attributes due to their rendering performance impact
+/plugins/services/src/js/components/forms/VolumesFormSection.tsx - Lambdas are forbidden in JSX attributes due to their rendering performance impact
```

Comment:
i currently consider disabling this rule. while it's a good practice to not have lambdas within JSX, so react can cache components for multiple rerenderings, they don't matter too much in most places.

DCOS-UI is mostly slow due to excessive JSON-parsing and the services table being implemented in a way that does not cache very much (it rerenders all the time).

Those lambdas are helping with readability a lot IMO. WDYT @janisz?